src-ui/lib/components/BooksTable.svelte

```diff
@@ -31,9 +39,9 @@
       qty: number
     }
   }
-  let promise: Promise<void> | undefined
+  let promise = $state<Promise<void>>()
```

Comment:
explicit `$state` as Svelte 5 expects instead of the previously implicit reactive state (from being at the top level of a component).